.dockerignore

```diff
@@ -17,3 +17,6 @@ www
 .terraform
 *.tfstate.backup
 terraform.tfstate.d
+
+rust/connlib/clients/android/connlib/build/
```

Comment:
These are also defined inside `rust/`, but Docker seems to only take the local context into account, so this was blowing up the `client` image when `docker compose build` was run locally.